concent-builder/test.yml

```diff
@@ -0,0 +1,37 @@
+- hosts:
+    - concent-builder
+  vars_files:
+    - consts.yml
+    - "{{ deployment_values }}/var.yml"
+    - "{{ deployment_values }}/var-{{ cluster }}.yml"
+  tasks:
+    - become:      yes
+      become_user: "{{ shared_user }}"
+      block:
+        - name: Configure kubectl to operate on the right cluster
+          command: gcloud container clusters get-credentials \
+              "{{ gke.cluster }}"                            \
+              --project "{{ gke.project }}"                  \
+              --zone    "{{ gke.zone }}"
+
+        - name:    Run custom network to allows communication between django and postgres containers
+          command: docker network create {{ image_prefix }}network
```

Comment:
Please use `{{ cluster }}` in network name.

I.e. `{{ cluster }}-{{ image_prefix }}unittest-network`

Reply:
Done.